### ISSUE TITLE
uTServer handler - Incorrect labeling with special characters

### ISSUE THREAD

**Issue by VLoub** [pymedusa/Medusa#9475]
**Describe the bug**
I am using uTorrent Server (uTServer) Labels for file destinations as it's intended but when the show is named with special characters such as question mark or a colon these characters are forwarded to Labels and then uTServer is unable to create these folders instead it creates names like "CCUEZH~B".

**To Reproduce**
1. Setup uTServer to use "Append the torrent's label"
2. Setup uTServer as a handler for torrents in Medusa
3. Use "Add label to torrent" with %N in Medusa
4. Add show with special characters in name such as "Case File Nº221: Kabukicho" or "So I'm a Spider, So What?"
5. Let it add to uTServer with labels
6. Wait for creation of folder for these downloads

**Expected behavior**
Creation of proper folder name.

**Intended Behavior**
Instead of labeling the show name (%N), could we append shows name from "Location" resp. path already trimmed of these characters (not a whole path but last leaf) because medusa is expecting it in this folder? I presume this is only reason to label torrents this way.

**Screenshots**
![image](https://user-images.githubusercontent.com/34989648/115951825-2332df00-a4e3-11eb-9c46-3d3e558081d2.png)

![image](https://user-images.githubusercontent.com/34989648/115952234-19aa7680-a4e5-11eb-9704-1fa0b268f5b9.png)

![image](https://user-images.githubusercontent.com/34989648/115951916-963c5580-a4e3-11eb-9c01-7aa5fa2b1150.png)

**Medusa:**
- OS: Synology / Docker Container
- Branch: master
- Commit: fcb5a1969dcd4ea0b6fb4a9debbf60179b0bbf4b
- Version: 0.5.11
- Database: 44.18

**Debug logs:**
Not needed - its not error generated in medusa.

Thanks for all your do regardless!

**Comment by VLoub** [comment]
So I added line into file 
https://github.com/pymedusa/Medusa/blob/3e258d2932fcc90437751efc3c5c4b82d447f3c5/medusa/clients/torrent/utorrent.py#L136-L143

``` 
re.sub("[:'!~?°;$&*#@%]", '', torrent_new_label)
```

It's not a fix and I don't think its for PR but for anyone facing the same, this could help. 
As mentioned I am running it in docker so after this I just mapped new file to 
```
-v ./utorrent.py:/app/medusa/medusa/clients/torrent/utorrent.py
```

**Comment by p0psicles** [comment]
Yeah I think a simple sanitation of the label name should be sufficient. I'll create a similar PR.
